gtsfm/two_view_estimator.py

```diff
@@ -484,111 +485,3 @@ def compute_relative_pose_metrics(
         return (None, None)
 
     return (R_error_deg, U_error_deg)
-
-
-def aggregate_frontend_metrics(
```

Comment:
What's happening here? Do we no longer need this or was it moved?

Reply:
I moved it to the utils file.